judgments/feeds.py

```diff
@@ -147,8 +147,11 @@ def item_description(self, item) -> str:
     def item_title(self, item):
         return item.name
 
+    def item_guid(self, item) -> str:
```

Comment:
Is this an appropriate name? It's not a guid?

Reply:
It's the name of a Django magic method for "thing which I use to get the `atom:id` element. I agree it's not well named at all, but the entire [syndication feed framework](https://docs.djangoproject.com/en/5.2/ref/contrib/syndication/) is a bit messy.

Reply:
oh yeah interesting their docs has this example: `    feed_guid = "/foo/bar/1234"  # Hard-coded guid.
z`

of what a guid is to them.... all good then